packages/localizations/src/en-US.ts

```diff
@@ -535,6 +535,8 @@ export const enUS: DeepRequired<LocalizationResource> = {
     form_code_incorrect: '',
     form_password_incorrect: '',
     not_allowed_access: '',
+    form_identifier_exists: '',
+    form_identifier_exists__username: '',
```

Comment:
❓ Why does `form_identifier_exists__username` exist only on `en-US` ?

Reply:
It doesn't matter - but it shouldn't. Fixing